src/include/crc32c.h

```diff
@@ -26,7 +40,13 @@ extern ceph_crc32c_func_t ceph_choose_crc32(void);
  */
 static inline uint32_t ceph_crc32c(uint32_t crc, unsigned char const *data, unsigned length)
 {
+  if (!data && length > 16)
```

Comment:
@kestrels On my test machine, the best point to fallback to regular was 16. I have difficulties in believing that anything more then 64 will be faster.